dockerd-entrypoint.sh

```diff
@@ -13,7 +15,7 @@ fi
 
 if [ "$1" = 'dockerd' ]; then
 	# if we're running Docker, let's pipe through dind
-	set -- "$(which dind)" "$@"
+	[ -x "$DIND" ] && set -- "$DIND" "$@"
```

Comment:
I don't see a strong reason for an extra variable here, so I'd rather see something even more straightforward like this:

```suggestion
	if [ -x /usr/local/bin/dind ]; then
		# if we have the (mostly defunct now) Docker-in-Docker wrapper script, use it
		set -- /usr/local/bin/dind "$@"
	fi
```

Reply:
My preference is for DRY as that tends to make future refactoring easier but as you like.